### DISCUSSION TITLE
Folder path for random choice?

### DISCUSSION THREAD

**Discussion by semi710** [nix-community/stylix#472] [Q&A]
is it possible to point to a folder and then a random image is picked on basis of that?
and a fallback image is chose if picking fails somehow.
and setting first frame if the chosen one is a gif?

**Comment by trueNAHO** [comment]
> is it possible to point to a folder and then a random image is picked on basis of that? and a fallback image is chose if picking fails somehow. and setting first frame if the chosen one is a gif?

This is [currently](https://github.com/danth/stylix/tree/29148118cc33f08b71058e1cda7ca017f5300b51) not possible.

AFAIK, https://github.com/danth/stylix/pull/102 is mostly abandoned and superseded by smaller and more managable tasks like #63 and #442.
